site/pages/competencies/junior.html

```diff
@@ -0,0 +1,11 @@
+---
+title: Junior Engineer Competencies
+description: Engineering competencies for Junior Engineers.
+permalink: /competencies/junior/
+layout: o-layout-docs
+---
+
+
+<h1 id="mid">Junior Engineer</h1>
```

Comment:
Oh I just spotted this, it should have an `id` of `junior`

```suggestion
<h1 id="junior">Junior Engineer</h1>
```